requirements.txt

```diff
@@ -1,6 +1,6 @@
 GitPython
 aicli
-PyYAML==5.4.1
+PyYAML
```

Comment:
are you saying that pip3.11 installs at least 5.4.1?

Reply:
Yes PyYAML 6.0.1, and more importantly trying to install 5.4.1 with pip3.11 breaks automation.